android/src/com/mapswithme/maps/editor/EditorFragment.java

```diff
@@ -235,41 +191,73 @@ private void refreshOpeningTime()
-    mEmailBlock = view.findViewById(R.id.block_email);
-    mEtEmail = findInput(mEmailBlock);
-    mCuisineBlock = view.findViewById(R.id.block_cuisine);
-    mCuisineBlock.setOnClickListener(this);
-    mTvCuisine = (TextView) view.findViewById(R.id.cuisine);
-    mWifiBlock = view.findViewById(R.id.block_wifi);
-    mSwWifi = (SwitchCompat) view.findViewById(R.id.sw__wifi);
-    mWifiBlock.setOnClickListener(this);
-    mOpeningHoursBlock = view.findViewById(R.id.block_opening_hours);
-    mEditOpeningHours = mOpeningHoursBlock.findViewById(R.id.edit_opening_hours);
+    mStreet = (TextView) view.findViewById(R.id.street);
+    View blockHouseNumber = view.findViewById(R.id.block_building);
+    mHouseNumber = findInputAndInitBlock(blockHouseNumber, 0, R.string.house_number);
+    mInputHouseNumber = (TextInputLayout) blockHouseNumber.findViewById(R.id.custom_input);
+    View blockZipcode = view.findViewById(R.id.block_zipcode);
+    mZipcode = findInputAndInitBlock(blockZipcode, 0, R.string.editor_zip_code);
+    mBlockLevels = view.findViewById(R.id.block_levels);
+    // TODO set levels limit (25 or more, get it from the core)
+    mBuildingLevels = findInputAndInitBlock(mBlockLevels, 0, R.string.editor_storey_number);
+    // Details
+    View blockPhone = view.findViewById(R.id.block_phone);
+    mPhone = findInputAndInitBlock(blockPhone, R.drawable.ic_phone, R.string.phone);
+    View blockWeb = view.findViewById(R.id.block_website);
+    mWebsite = findInputAndInitBlock(blockWeb, R.drawable.ic_website, R.string.website);
+    View blockEmail = view.findViewById(R.id.block_email);
+    mEmail = findInputAndInitBlock(blockEmail, R.drawable.ic_email, R.string.email);
+    View blockCuisine = view.findViewById(R.id.block_cuisine);
+    blockCuisine.setOnClickListener(this);
+    mCuisine = (TextView) view.findViewById(R.id.cuisine);
+    View blockOperator = view.findViewById(R.id.block_operator);
+    mOperator = findInputAndInitBlock(blockOperator, R.drawable.ic_operator, R.string.editor_operator);
+    View blockWifi = view.findViewById(R.id.block_wifi);
+    mWifi = (SwitchCompat) view.findViewById(R.id.sw__wifi);
+    blockWifi.setOnClickListener(this);
+    View blockOpeningHours = view.findViewById(R.id.block_opening_hours);
+    mEditOpeningHours = blockOpeningHours.findViewById(R.id.edit_opening_hours);
     mEditOpeningHours.setOnClickListener(this);
-    mEmptyOpeningHours = mOpeningHoursBlock.findViewById(R.id.empty_opening_hours);
+    mEmptyOpeningHours = blockOpeningHours.findViewById(R.id.empty_opening_hours);
     mEmptyOpeningHours.setOnClickListener(this);
-    mOpeningHours = (TextView) mOpeningHoursBlock.findViewById(R.id.opening_hours);
+    mOpeningHours = (TextView) blockOpeningHours.findViewById(R.id.opening_hours);
     mOpeningHours.setOnClickListener(this);
+    mDescription = findInput(view.findViewById(R.id.cv__more));
+
+    mMetaBlocks.append(MetadataType.FMD_OPEN_HOURS.toInt(), blockOpeningHours);
```

Comment:
? Мы пока всё ещё возвращаем 
GetEditableFields()


> On Apr 21, 2016, at 3:56 PM, Alex Zolotarev <notifications@github.com> wrote:
> 
> In android/src/com/mapswithme/maps/editor/EditorFragment.java <https://github.com/mapsme/omim/pull/2947#discussion_r60574202>:
> 
> >      mOpeningHours.setOnClickListener(this);
> > +    mDescription = findInput(view.findViewById(R.id.cv__more));
> > +
> > +    mMetaBlocks.append(MetadataType.FMD_OPEN_HOURS.toInt(), blockOpeningHours);
> GetEditableProperties() работает с другим енамом, его использовать - не?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/2947/files/44b3f477a158544a38212dbfd95100ddbb3c305c#r60574202>